src/services/gapi.js

```diff
@@ -0,0 +1,36 @@
+/* global gapi */
+import $S from 'scriptjs';
+import once from 'lodash/once';
+import config from '../config';
+
+export const loadGapi = once(async() => new Promise((resolve) => {
+  $S('https://apis.google.com/js/client.js', async() => {
+    resolve(gapi);
+  });
+}));
+
+export const getGapi = once(async() => {
```

Comment:
It seems like maybe `loadAndConfigureGapi()` is a more precise description of what this function is doing?

Reply:
changed